### DISCUSSION TITLE
Can't access my flickr images

### DISCUSSION THREAD

**Discussion by grunta56** [mcguirepr89/BirdNET-Pi#386] [Q&A] [answered]
Hi
I have the API installed and BirdnetPi downloads numerous publicly available images. Some of these are not very good so I want to start my own library of images but I am having trouble linking to my own flickr account using the email address that is attached to my account together with the API. I put images in my flickr photo stream and they are "public" but no images are downloading. I'm not sure what I'm missing here. Having said that I have not used flickr before. Any assistance appreciated. Thank you, Grant

**Comment by ehpersonal38** [comment]
Hi there - sorry to hear you're having problems with the Flickr email filter. I did just recently implement this so I wouldn't be surprised if there's a bug or two. I think it'd be easiest if you'd supply me with your Flickr email so I can run some troubleshooting with my installation. So you can either paste it here or email me `ehpersonal38@gmail.com` and I'll see what I can find.

**Comment by ehpersonal38** [comment, marked as answer]
Hi again grant,

I discovered the issue this morning and have pushed a fix. You can go to Tools -> System Controls -> Update to get the latest files. You will need to clear your cookies for BirdNET-Pi, which you can do on Chrome by copying and pasting this into a new tab URL, and clicking the trash can next to birdnetpi. 
`chrome://settings/siteData?searchSubpage=birdnet&search=cookies`

After that you should be seeing your images as intended! Let me know if this works for you or doesn't. 

Ethan 🐤

**Comment by grunta56** [comment]
Hi Ethan



thank you very much! Its working



Best regards

Grant

 




---- On Sat, 18 Jun 2022 22:10:29 +1000 ehpersonal38 ***@***.***> wrote ----





Hi again grant,

I discovered the issue this morning and have pushed a fix. You can go to Tools -> System Controls -> Update to get the latest files. You will need to clear your cookies for BirdNET-Pi, which you can do on Chrome by copying and pasting this into a new tab URL, and clicking the trash can next to birdnetpi.
 chrome://settings/siteData?searchSubpage=birdnet&search=cookies
After that you should be seeing your images as intended! Let me know if this works for you or doesn't.

Ethan 🐤
